### PR TITLE
BaSP-MR-64: subscription add edit form

### PR DESCRIPTION
**Pull request by ghost**
Subscription add edit form.

My colleagues were fundamental participants in this pull request.
We lost 5 hours because last week's back end was poorly done.
However, it is important to remember that we did not have the capacity to detect that the populate was wrong, an assistant had to come.

Daniel, Gonzalo, Federico, Emanuel, Ale M, Ale S, Lucho (auxiliary)

### CODE REVIEW

**Comment by fernandofernandezuy** [comment]
[[Subscription] Add/Edit Form](https://trello.com/c/Qjh2JJDT/64-subscription-add-edit-form)

File: src/Components/Subscriptions/Table/index.jsx [hunk 1]
```diff
@@ -3,10 +3,13 @@ import styles from './table.module.css';
 import { FaEdit } from 'react-icons/fa';
 import { FaTimes } from 'react-icons/fa';
```
Comment (EmaBaldor) [inline src/Components/Subscriptions/Table/index.jsx:4]: You can import them in the same line.

File: src/Components/Subscriptions/index.jsx [hunk 2]
```diff
@@ -1,15 +1,51 @@
...
+      date: new Date()
+    };
+
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/subscriptions`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(newSubscription)
```
Comment (EmaBaldor) [inline src/Components/Subscriptions/index.jsx:30]: You can add confirmation messages of the actions.

**Review by LucianoAlarcon — APPROVED** [review]

**Review by lorenzomauro59 — APPROVED** [review]

**Comment by fernandofernandezuy** [comment]
[[HOTFIX] - React router version and delete activities](https://trello.com/c/r7VIPbxs/91-hotfix-react-router-version-and-delete-activities)

**Pull request merged**